docs/source/guides/worklock_guide.rst

```diff
@@ -52,12 +324,153 @@ You can obtain information about the current state of WorkLock by running:
+    ------------------------------------------------------
+    Base Deposit Rate ..... 1000 NU per base ETH
+
+    Bonus (surplus over minimum bid)
+    ------------------------------------------------------
+    Bonus ETH Supply ...... 35000 ETH
+    Bonus Lot Size ........ 265000000 NU
+    Bonus Deposit Rate .... 7571.43 NU per bonus ETH
+
+    Refunds
+    ------------------------------------------------------
+    Refund Rate Multiple .. 4.00
+    Bonus Refund Rate ..... 1892.86 units of work to unlock 1 bonus ETH
+    Base Refund Rate ...... 250.0 units of work to unlock 1 base ETH
+
+        * NOTE: bonus ETH is refunded before base ETH
+
+
+For the less obvious values in the output, here are some definitions:
+
+    - Lot Size
+        NU tokens to be distributed by WorkLock
+    - ETH Supply
+        Sum of all ETH bids that have been placed
+    - ETH Pool
+        Current ETH balance of WorkLock that accounts for refunded ETH for whales i.e. `ETH Supply` - `Whale Refunds`
+    - Refund Rate Multiple
+        Indicates how quickly your ETH is unlocked relative to the deposit rate e.g. a value of ``4`` means that you get your ETH refunded 4x faster than the rate used when you received NU
+    - Base Deposit Rate
+        Amount of NU to be received per base ETH in WorkLock
+    - Bonus ETH Supply
+        Sum of all ETH bonus bids that have been placed i.e. sum of all ETH above minimum bid
+    - Bonus Lot Size
+        Amount of NU tokens tokens that are available to be distributed based on the bonus part of bids
+    - Bonus Deposit Rate
+        Amount of NU to be received per bonus ETH in WorkLock
+    - Bonus Refund Rate
+        Units of work to unlock 1 bonus ETH
+    - Base Refund Rate
+        Units of work to unlock 1 base ETH
+
+
+If you want to see specific information about your current bid, you can specify your bidder address with the ``--bidder-address`` flag:
 
 .. code::
 
     (nucypher)$ nucypher worklock status --bidder-address <YOUR BIDDER ADDRESS> --network <NETWORK> --provider <YOUR PROVIDER URI>
 
+The following output is an example of what is included when ``--bidder-address`` is used
+
+.. code::
+
+    WorkLock Participant <BIDDER ADDRESS>
+    =====================================================
+    Tokens Claimed? ...... No
+    Total Bid ............ 22 ETH
+        Base ETH ......... 15 ETH
+        Bonus ETH ........ 7 ETH
+    Tokens Allocated ..... 68000 NU
+
+    Completed Work ....... 0
+    Available Refund ..... 0 ETH
+
+    Refunded Work ........ 0
+    Remaining Work ....... <>
+
+Alternatively, when the allocated tokens have been claimed, the following is an example of the output
+
+.. code::
+
+    WorkLock Participant <BIDDER ADDRESS>
+    =====================================================
+    Tokens Claimed? ...... Yes
+    Current Locked ETH ... 22 ETH
+
+    Completed Work ....... 0
+    Available Refund ..... 0 ETH
+
+    Refunded Work ........ 0
+    Remaining Work ....... <>
```

Comment:
Looks weird, because after claiming this value won't be zero

Reply:
✔️ - I had to manufacture this output, and didn't know what to put there - I'll add something generic